concepticondata/conceptlists/Morucci-2019-643.tsv

```diff
@@ -0,0 +1,644 @@
+Morucci-2019-643-606	606	udibile	audible	0.028	0.028	0.389	4.972	1.139	4.972	udito	6.556	0.728	0.754	3.833	4.95	5.211		
+Morucci-2019-643-607	607	ululante	howling	0	0	0.189	5	2.541	5	udito	7.73	0.738	0.647	4.353	4.444	4.944		
+Morucci-2019-643-608	608	umidiccio	moist	0.95	1.725	4.725	0.625	3.725	4.725	tatto	11.75	1.372	0.349	4.474	4.294	5.053	143	HUMID
+Morucci-2019-643-609	609	umido	moist	1.553	2.289	4.316	0.263	3.514	4.316	tatto	11.935	1.394	0.34	6.824	3.944	6.556		
+Morucci-2019-643-610	610	universale	universal	0.784	0.73	0.784	2.081	2.405	2.405	vista	6.784	1.469	0.247	6.444	5.2	5.789		
+Morucci-2019-643-611	611	unto	oily	3.459	1.722	4.676	0.27	3.943	4.676	tatto	14.07	1.401	0.313	6.167	4	5.895		
+Morucci-2019-643-612	612	untuoso	oily	3.424	1.848	4.515	0.576	3.939	4.515	tatto	14.303	1.455	0.275	3.111	5.45	3.421		
+Morucci-2019-643-613	613	urlante	screamer	0	0	0.081	4.972	2.722	4.972	udito	7.776	0.701	0.639	4.444	3.95	4.842		
+Morucci-2019-643-614	614	urticante	stinging	1.065	0.75	4.844	0.281	2.562	4.844	tatto	9.502	1.247	0.48	3.111	5.25	4.316		
+Morucci-2019-643-615	615	ustionante	burning	1.579	1.179	4.795	0.923	3.513	4.795	tatto	11.989	1.419	0.323	4.667	5	5.684		
+Morucci-2019-643-616	616	vago	vague	1	1.125	0.606	2.848	2.364	2.848	udito	7.943	1.462	0.282	5.947	4.471	4.789		
+Morucci-2019-643-617	617	variopinto	colorful	0.462	0.308	0.359	0.385	4.949	4.949	vista	6.462	0.866	0.718	5.294	4.412	5.389	2997	COLORFUL
+Morucci-2019-643-618	618	vasto	vast	0.03	0.333	2.242	1.758	4.939	4.939	vista	9.303	1.132	0.528	5.947	4.235	5.842		
+Morucci-2019-643-619	619	vegetale	vegetable	3.615	2.897	3.128	0.487	4.667	4.667	vista	14.795	1.469	0.283	6.278	3.85	6.263		
+Morucci-2019-643-620	620	velato	veiled	0.706	0.735	2.765	1.706	4.088	4.088	vista	10	1.402	0.338	4.579	4.941	4.789		
+Morucci-2019-643-621	621	vellutato	velvety	1.41	0.667	4.395	0.684	4	4.395	tatto	11.156	1.336	0.334	4.789	4.235	5.579		
+Morucci-2019-643-622	622	venato	veined	0.289	0.053	3.105	0.263	4.421	4.421	vista	8.132	0.961	0.537	3.158	5.933	4.263		
+Morucci-2019-643-623	623	ventilato	ventilated	0.128	1.538	3.333	2.692	3.333	3.333	tatto	11.026	1.394	0.291	5.895	4.471	6.211		
+Morucci-2019-643-624	624	verdastro	greenish	0.353	0.5	0.324	0.206	4.853	4.853	vista	6.235	0.826	0.745	4.647	4.278	5.056		
+Morucci-2019-643-625	625	verde	green	0.475	0.4	0.5	0.125	5	5	vista	6.5	0.838	0.75	6.882	2.333	6.611	1425	GREEN
+Morucci-2019-643-626	626	verdeggiante	verdant	0.189	0.838	0.378	0.324	4.838	4.838	vista	6.568	0.903	0.708	4.059	4.118	5.5		
+Morucci-2019-643-627	627	verdino	verdino	0.088	0.147	0.176	0.059	4.971	4.971	vista	5.441	0.407	0.903	3.947	3	4.316		
+Morucci-2019-643-628	628	vermiglio	vermilion	0.194	0.167	0.333	0.111	4.917	4.917	vista	5.722	0.59	0.84	2.421	5.412	3.526		
+Morucci-2019-643-629	629	vero	TRUE	1.486	1.389	1.973	2.973	3.757	3.757	vista	11.578	1.534	0.205	7	2.647	5.368	1657	TRUE
```

Comment:
the English translation has capitals here, probably read as a boolean in Excel/OO